pkg/controller/core/workload_controller.go

```diff
@@ -531,3 +607,75 @@ func (h *resourceUpdatesHandler) queueReconcileForPending(ctx context.Context, _
 		}
 	}
 }
+
+type workloadCqHandler struct {
+	client client.Client
+}
+
+var _ handler.EventHandler = (*workloadCqHandler)(nil)
+
+// Create is called in response to an create event - e.g. Pod Creation.
```

Comment:
Why the comment mentions Pods (here and below)?

Reply:
`impl` behavior,  removed